bridge/src/org/sparkboard/promise.cljc

```diff
@@ -0,0 +1,60 @@
+(ns org.sparkboard.promise
+  (:refer-clojure :exclude [let -> ->> do resolve try catch])
+  #?(:cljs (:require kitchen-async.promise
+                     [clojure.walk :as walk])
+     :clj  (:require [net.cgrand.macrovich :as m]
+                     [clojure.walk :as walk]))
+  #?(:cljs
+     (:require-macros [net.cgrand.macrovich :as m]
+                      kitchen-async.promise
+                      org.sparkboard.promise)))
+
+(m/deftime
+  (defmacro alt [{:keys [cljs clj]} & body]
```

Comment:
Why have these as `.cljc` at all?

Reply:
There are a number of places  where clj blocks, and cljs returns promises. the kitchen-async macros let us work with promises as if they block. by having p/let defer to core/let in clj, we can avoid using reader conditionals and copy/pasting code that would be otherwise identical